### ISSUE TITLE
v0.8.0 issue with variable

### ISSUE THREAD

**Issue by yuriydzobak** [VictoriaMetrics/grafana-datasource#150]
Hi, 
I upgraded the plugin from version 0.7.0 to 0.8.0, and now all dashboards display messages in a specific format.
![image](https://github.com/VictoriaMetrics/grafana-datasource/assets/70581130/7fd08bcc-3d6f-45ba-a70a-3a3af20a9112)

The graphs are loading, but when you go into settings and variables, they appear in a new fancy format as shown in this PR. https://github.com/VictoriaMetrics/grafana-datasource/issues/144 
![image](https://github.com/VictoriaMetrics/grafana-datasource/assets/70581130/747f5f11-7202-42d5-a242-13e8ba28be23)

It converted an old type to a new one, but it doesn't return any data or display alerts. Also, all variable selectors are broken. 
You verify, when you have version 0.7.0 and any dashboard with variables then upgrade plugin to 0.8.0 and it cause the same issue

**Comment by Loori-R** [comment]
Hi, @yuriydzobak.
Can you check the browser console for a more detailed error message?

**Comment by yuriydzobak** [comment]
> Hi, @yuriydzobak. Can you check the browser console for a more detailed error message?

Hi,
Log file
[grafana-1712813714051.log](https://github.com/VictoriaMetrics/grafana-datasource/files/14941796/grafana-1712813714051.log)
How looks
![image](https://github.com/VictoriaMetrics/grafana-datasource/assets/70581130/e4715e31-4b9c-44a8-a481-90440bf0b535)
![image](https://github.com/VictoriaMetrics/grafana-datasource/assets/70581130/0889049d-9555-4bad-9d6a-d5cd5dfb50c2)

let me know if you need something too
Thanks

**Comment by dg-nvm** [comment]
if this helps, we updated our Grafana plugin to 0.8.0 and no developer reported any new issue. Grafana 10.4.0

**Comment by yuriydzobak** [comment]
The grafana version is `v10.1.5`

**Comment by Loori-R** [comment]
Indeed, this error does not occur in newer versions (in `10.3.1` the error does not reproduce). I opened a [PR #153](https://github.com/VictoriaMetrics/grafana-datasource/pull/153) and with this change, it should work on any version.
